pkg/secrets/nop.go

```diff
@@ -0,0 +1,13 @@
+package secrets
+
+type nopDecrypter struct {
```

Comment:
What's the purpose of this?

Reply:
testing via a secret engine that just returns the content.